airbyte_cdk/test/standard_tests/models/scenario.py

```diff
@@ -83,3 +96,29 @@ def __str__(self) -> str:
             return f"'{self.config_path.name}' Test Scenario"
 
         return f"'{hash(self)}' Test Scenario"
+
+    def without_expecting_failure(self) -> ConnectorTestScenario:
```

Comment:
This method and the below allow us to get a mutated instance of the scenario, while still treating each individual scenario instance as immutable.